Filter/StringFilter.php

```diff
@@ -55,7 +55,7 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
      *
      * @return bool
      */
-    private function getOperator($type)
+    protected function getOperator($type)
```

Comment:
Actually what I did in the link above is use strtolower on both operands, which is suboptimal because an index on the field cannot be used, and function based indexes in doctrine support is unsure. Related : https://github.com/doctrine/dbal/issues/980